x/slashing/client/cli/cli_test.go

```diff
@@ -3,9 +3,10 @@
 package cli_test
 
 import (
-	"github.com/cosmos/cosmos-sdk/x/slashing/client/testutil"
 	"testing"
 
+	"github.com/cosmos/cosmos-sdk/x/slashing/client/testutil"
+
 	"github.com/stretchr/testify/require"
 
```

Comment:
```suggestion

	"github.com/stretchr/testify/require"

	"github.com/cosmos/cosmos-sdk/x/slashing/client/testutil"       	
```

goimport doesnt pull down local all the way

Reply:
I think it's related to the fact that `goimports` is incapable of regrouping imports automatically